edit_datasets/edit_wms_dataset.py

```diff
@@ -0,0 +1,124 @@
+import os
+import requests
+import json
+
+API_TOKEN = os.getenv('RW_API_KEY')
+
+def create_headers():
+    return {
+        'content-type': "application/json",
+        'authorization': "{}".format(os.getenv('apiToken')),
```

Comment:
Use the same format as the other update? `'authorization': f'Bearer {API_TOKEN}'`